### PR TITLE
add peft_module_casting_to_bf16 in DPOTrainer

### PR DESCRIPTION
**Pull request by sywangyi**

### CODE REVIEW

**Comment by sywangyi** [comment]
@younesbelkada

**Review by younesbelkada — APPROVED** [review]
Thanks!

**Comment by younesbelkada** [comment]
@sywangyi can you run the styling checks? `make precommit`

File: trl/trainer/dpo_trainer.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 from ..import_utils import is_peft_available, is_wandb_available
 from ..models import PreTrainedModelWrapper, create_reference_model
-from .utils import DPODataCollatorWithPadding, disable_dropout_in_model, pad_to_length, trl_sanitze_kwargs_for_tagging
+from .utils import DPODataCollatorWithPadding, disable_dropout_in_model, pad_to_length, trl_sanitze_kwargs_for_tagging, peft_module_casting_to_bf16
```
Comment (kashif) [inline trl/trainer/dpo_trainer.py:44]:
```suggestion
from .utils import (
    DPODataCollatorWithPadding,
    disable_dropout_in_model,
    pad_to_length,
    peft_module_casting_to_bf16,
    trl_sanitze_kwargs_for_tagging,
)
```

**Review by kashif — APPROVED** [review]

**Comment by younesbelkada** [comment]
Thanks @kashif @sywangyi !

**Pull request merged**